context/app/api/vitessce_confs/fixtures/output_expected/multi_image_sprm_anndata_conf.json

```diff
@@ -0,0 +1,406 @@
+[
```

Comment:
If I understand it correctly, the files in `input_entity` and `output_expected` are parallel? Not essential, but it might be nice to rearrange the directory structure so that input/output pairs are next to each other, each directory containing just two files. Depending the name matching is a little fragile, it's possible for the lists to fall out of sync.

Reply:
It's sorted not parallel so they can come in any order but will always output in this order
```python
for id in sorted(found_ids):
```